neuracore/core/utils/server.py

```diff
@@ -32,6 +32,13 @@ class CheckpointRequest(BaseModel):
     epoch: int
 
 
+class PredictRequest(BaseModel):
+    """Request model for predicting."""
+
+    sync_point: SyncPoint
+    robot_name: Optional[str] = None
+
+
```

Comment:
This is also used in the neuracore package, so please move to neuracore_types.
This is something in general that needs to happen for any new types introduced.